### ISSUE TITLE
[Discussion] indexedDB for graphql cache

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#238]
### TLDR
Use indexedDB ([LocalForage](https://localforage.github.io/localForage/)) for graphql cache.

### Description
indexedDB is a better, less mutable low-level browser API with more space than localStorage (5mb)
[LocalForage](https://github.com/localForage/localForage) is a great framework for using indexedDB, with fallback to WebSQL and localStorage. 

[docs](https://localforage.github.io/localForage/)

For GraphQL/Apollo implementation, [apollo-cache-persist](https://github.com/apollographql/apollo-cache-persist) is a good option.
[blogpost about apollo cache persist](https://www.apollographql.com/blog/announcing-apollo-cache-persist-cb05aec16325)

### Why?
* If Did365 can be used as a PWA we have to use another mechanism than localStorage 
* Reports will possibly be very large (all users, all entries, whole year), and will not fit inside localStorage.
